contracts/extensions/extend/HumanboundExtendLogic.sol

```diff
@@ -18,7 +18,10 @@ contract HumanboundExtendLogic is ExtendLogic {
             emit OperatorInitialised(_lastCaller());
         }
 
-        require(_lastCaller() == state.operator, "HumanboundExtendLogic: unauthorised");
+        require(
+            _lastCaller() == state.operator || _lastCaller() == address(this),
```

Comment:
Just to confirm `_lastCaller() == address(this)` was added to allow calls to extend from another extension, such as HumanboundReplaceLogic?

Reply:
Exactly.